CentralityiOS/CentralityiOS/ViewControllers/UsageDashboardViewController.m

```diff
@@ -0,0 +1,177 @@
+//
+//  UsageDashboardViewController.m
+//  CentralityiOS
+//
+//  Created by Samuel Osa-Agbontaen on 8/11/22.
+//
+
+#import "UsageDashboardViewController.h"
+#import "Parse/Parse.h"
+#import "SceneDelegate.h"
+#import "Charts-Swift.h"
+#import "FSCalendar.h"
+#import "CentralityHelpers.h"
+#import "TaskObject.h"
+
+@interface UsageDashboardViewController ()<ChartViewDelegate>
+@property (weak, nonatomic) IBOutlet PieChartView *pieChart;
+@property NSMutableArray<CategoryObject*>* userCategories;
+@property (nonatomic, assign) BOOL shouldHideData;
+
+@end
+
+NSTimer* refreshTimer;
```

Comment:
same note about global timers. let me know if these are causing problems - should probably be inside of the viewController or some central object that is shared between viewControllers. (in our code we often have simple classes called `toolboxes` that contain things like this that need to live beyond individual viewControllers)